### PR TITLE
Change package name

### LINKED ISSUE #1: Bones needs a new name!
@star-lebec — don't you think "Bones" is a little too dry? Nyuk nyuk. Let's have a custom name!

### PR DESCRIPTION
**Pull request by star-lebec**
Closes #1.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "__BONES_NEEDS_A_NAME__",
+  "name": "dogbones",
```
Comment (glebec) [inline package.json]: You know what, on second thought, `dogbones` is too cute. Let's just do `spooky`. Because that's not cute at all.

**Review by glebec — CHANGES_REQUESTED** [review]
Looks pretty good, but let's just fix up that one detail.

**Comment by star-lebec** [comment]
OK I'll get right on that.

**Comment by star-lebec** [comment]
Hey @glebec can you check it now?

**Review by glebec — APPROVED** [review]

**Comment by glebec** [comment]
Awesome, let's do it!

**Pull request merged**